app/Domain/Timesheets/Templates/showAll.blade.php

```diff
@@ -132,38 +132,35 @@
                         </select>
                     </td>
                     <td>
-                        <x-global::forms.checkbox
-                            labelText="{{ __('label.invoiced') }}"
-                            labelPosition="right"
-                            name="invEmpl"
-                            id="invEmpl"
-                            value="on"
-                            :checked="$tpl->get('invEmpl') == '1'"
-                            onclick="submit();"
+                        <input type="checkbox" value="on" name="invEmpl" id="invEmpl" onclick="submit();"
```

Comment:
That needs to be reverted. Pull from the dev branch first

Reply:
This is true for all checkboxes. Did you do a hard overwrite?

Reply:
reverted